wcivf/apps/elections/templates/elections/fptp.html

```diff
@@ -1,99 +1,101 @@
+                        seats are multi-member, meaning, for example,
+                        that they may return three councillors to represent
+                        one ward. In this instance, the voter will get three
+                        votes. This is sometimes called a block vote.
+                        Voters do not have to vote for candidates from the
                         same party, however.
                     </p>
-                        
+
                     <h3>Example election</h3>
-                    <p>Taking an example ward from the 2019 local elections 
-                        in Lincolnshire, you can see that the independent 
-                        candidate was the most popular and so was elected, 
-                        but only two of the three Conservatives were. If the 
-                        Conservative vote had been split more evenly between 
-                        their candidates, then they may have allowed the sole 
-                        Labour candidate to get elected. Parties and voters, 
-                        therefore, have to take an educated guess at how well 
-                        they may do in this sort of election. Parties may wish 
+                    <p>Taking an example ward from the 2019 local elections
+                        in Lincolnshire, you can see that the independent
+                        candidate was the most popular and so was elected,
+                        but only two of the three Conservatives were. If the
+                        Conservative vote had been split more evenly between
+                        their candidates, then they may have allowed the sole
+                        Labour candidate to get elected. Parties and voters,
+                        therefore, have to take an educated guess at how well
+                        they may do in this sort of election. Parties may wish
                         to indicate who their lead candidates are to voters.
-                    </p>  
+                    </p>
                     <div class="ds-table">
-                <table>
-                    <caption>
-                        <h2>
-                            2019 South Kesteven District Council election
-                        </h2>
-                        <h2>
-                            Deeping St James Ward (three seats) 
-                        </h2>
-                    </caption>
-                    <tbody>
-                        <tr>
-                            <th>Candidates</th>
-                            <th>Votes</th>
+                        <table>
+                            <caption>
+                                <h2>
+                                    2019 South Kesteven District Council election
+                                </h2>
+                                <h2>
+                                    Deeping St James Ward (three seats)
```

Comment:
Think this line is repeated three times by mistake?

Reply:
Weird. Thanks for catching.